{% extends "basic_layout.html" %}

{% block link_here %}
  <link rel="stylesheet" href="{{ url_for('static',filename='css/home.css') }}">
{% endblock link_here %}

{% block content %}
    <div class="about_div row justify-content-center my-4">
      <div class="about_header col-2">
        <h2>About</h2>
      </div>
      <div class="info_image_div col-2">
        <img class="info_image_icon" src="{{ url_for('static',filename='img/information.png') }}" alt="Info Image">
      </div>
    </div>
    <div class="about-content">
      <p>
        <u><strong>Gen</strong></u>omic <u><strong>Di</strong></u>stribution of structural <u><strong>S</strong></u>uperfamilies or <strong>GenDiS</strong> is a repository of homologous sequences of structural superfamily members in SCOP identified in the sequence space (NR, NCBI).
      </p>
      <p>
         This version of GenDiS+ is in direct correspondence with SCOP 1.75, PASS 2.4, NR, and UniProt. We used remote homology methods like PHI-BLAST and CS-BLAST for identifying the homologues of over 11,000 SCOP members from 1961 superfamilies (Iyer M, Joshi A and Sowdhamini R, 2018). Hits were validated using the structure-based sequence alignments for superfamilies as provided in PASS2.4. 18,325,278 hits were identified by sequence searches from 67,289,356 (27.2%) sequences in NR, NCBI; out of which 14,495,466 (21.5%) sequences passed the validation. Using all the proteins predicted to have at least one structural domain, a high coverage of 61% of Pfam families was achieved which is the higher than the existing methods.
      </p>
      <p>
        	The database reports the analysis of the hits obtained at the superfamily, fold and class- level, the domain architecture (DA), and the taxonomic occurrence. Compared to the previous version (Pugalenthi G, Bhaduri A and Sowdhamini R, 2005), we have now computed DA of the hits at the Pfam level, identified correspondence between Pfam and SCOP domain definitions of the hits, and classified all the Pfam and SCOP DA for each superfamily hits.
      </p>
      <h4>The user may navigate the database at different levels:</h4>
      <ol>
        <li>SCOP hierarchies- SCOP class, fold and superfamily code and description</li>
        <li>Taxonomic hierarchies- NCBI taxid and description</li>
        <li>SCOP and Pfam domain architecture- based on SCOP superfamily code/description and Pfam family code/description</li>
      </ol>
      <h4 class="h4">If you found our database or the methodology useful please cite us:</h4>
      <ul>
        <li>Iyer MS, Joshi AG and Sowdhamini R. Genome-wide survey of remote homologues for protein domain superfamilies of known structure reveals unequal distribution across structural classes. Mol. Omics, 2018,14, 266-280</li>
        <li>Joshi AG, Raghavender US and Sowdhamini R. Improved performance of sequence search approaches in remote homology detection. F1000Research 2014, 2:93</li>
        <li>Pugalenthi, G. Bhaduri, A and Sowdhamini, R (2005) GenDiS: Genomic Distribution of Protein Structural domain Superfamilies. Nucleic Acid Research Vol. 33, D252-D255</li>
      </ul>
      <p style="margin-top: 60px">
        	Please contact us for further clarifications and for running the pipeline on multiple sequences.
      </p>
    </div>
    <div class="statistics">
      <div class="statistics_header row justify-content-center mb-5">
        <div class="stats_header col-3">
          <h2>Statistics</h2>
        </div>
        <div class="stats_image col-1">
          <img class="stats_icon" src="{{ url_for('static',filename='img/statistics_header_image.png') }}" alt="statistics image">
        </div>
      </div>
      <div class="statistics_table">
        <table class="stats_table">
          <thead>
            <tr>
              <th class="cell_head" >Database</th>
              <th class="cell_head" >Level</th>
              <th class="cell_head" >Total</th>
              <th class="cell_head" >Covered</th>
              <th class="cell_head" >%Coverage</th>
            </tr>
          </thead>
          <tbody>
            <tr>
              <td class="cell" >Pfam</td>
              <td class="cell" >Families</td>
              <td class="cell" >16230</td>
              <td class="cell" >9853</td>
              <td class="cell" >60.71</td>
            </tr>
            <tr>
              <td class="cell" >SCOP v1.75</td>
              <td class="cell" >Superfamilies</td>
              <td class="cell" >1962</td>
              <td class="cell" >1961</td>
              <td class="cell" >99.94</td>
            </tr>
            <tr>
              <td class="cell" >SCOPe v2.06</td>
              <td class="cell" >Superfamilies</td>
              <td class="cell" >2008</td>
              <td class="cell" >1961</td>
              <td class="cell" >97.66</td>
            </tr>
            <tr>
              <td class="cell" >NR db</td>
              <td class="cell" >Sequences</td>
              <td class="cell" >67,289,356</td>
              <td class="cell" >18,325,278</td>
              <td class="cell" >27.2</td>
            </tr>
            <tr>
              <td class="cell" >PDB</td>
              <td class="cell" >Protein structures</td>
              <td class="cell" >130,536</td>
              <td class="cell" >81,026</td>
              <td class="cell" >62.07</td>
            </tr>
            <tr>
              <td class="cell" >UniProt</td>
              <td class="cell" >Sequences</td>
              <td class="cell" >28,827,044</td>
              <td class="cell" >9,730,213</td>
              <td class="cell" >29.62</td>
            </tr>
            <tr>
              <td class="cell" >Taxonomy db, NCBI</td>
              <td class="cell" >Organisms</td>
              <td class="cell" >164,890</td>
              <td class="cell" >67,377</td>
              <td class="cell" >40.86</td>
            </tr>
            <tr>
              <td class="cell" >SwissProt</td>
              <td class="cell" >Sequences</td>
              <td class="cell" >557,275</td>
              <td class="cell" >298,699</td>
              <td class="cell" >53.60</td>
            </tr>
          </tbody>
        </table>
      </div>
    </div>
{% endblock content %}
